src/styles/product_card.scss

```diff
@@ -0,0 +1,112 @@
+.product {
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  transition: all 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &_img {
+    display: flex;
+    justify-content: center;
+    margin-inline: auto;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &_name {
+    font-size: 12px;
+    line-height: 18px;
+    font-family: $mainFont;
+    font-weight: 500;
+    color: #060b35;
+    margin: 0 0 4px;
+
+    &:hover {
+      color: #34568b;
+    }
+  }
+
+  &_code {
+    font-size: 10px;
+    line-height: 14px;
+
+    font-family: $mainFont;
+    font-weight: 400;
+    color: rgba(97, 96, 112, 1);
+    margin: 0;
+  }
+
+  &_rating {
+    padding-block: 16px 24px;
+    display: flex;
+    justify-content: space-between;
+  }
```

Comment:
The `padding-block: 16px 24px;` rule is not valid. The `padding-block` property sets the block start and end paddings of an element, which correspond to the top and bottom paddings for a horizontally written element. If you want to set vertical and horizontal padding, use `padding: 16px 24px;` instead.